web/src/pages/stacklets/list.tsx

```diff
@@ -76,3 +80,19 @@ const StackletCondition = (props: { condition: DisplayCondition }) => (
     {props.condition.condition}
   </span>
 );
+
+const StackletEndpoints = (props: {
+  endpoints: { [key: string]: string | undefined };
+}) => (
+  <For each={Object.entries(props.endpoints)}>
+    {(item) => (
+      <ul class='p-0 m-0'>
```

Comment:
This is not the final solution for displaying these endpoints. However, we don't need a list for each endpoint. Instead we should create one list and multiple `<li>` elements; one for each endpoint.

Reply:
Good point, fixed in 7c9377106984422b4eec51a3e45e9b0294c4aba0